### ISSUE TITLE
Auth/unauthorized-domain when running project locally

### ISSUE THREAD

**Issue by marcinwmichna** [jrobind/dev-jot#65]
When running locally i get this message: 
auth/unauthorized-domain This domain (127.0.0.1) is not authorized to run this operation. Add it to the OAuth redirect domains list in the Firebase console -> Auth section -> Sign in method tab.

Cannot Sign in with Google.

![image](https://user-images.githubusercontent.com/49107774/102206917-968d6a00-3ecd-11eb-9c0e-d5daef152880.png)

**Comment by buigabor** [comment]
I ran into the same problem, after trying this it worked for me:

If you run into an error stating "auth/unauthorized domain This domain (127.0.0.1) is not authorized to run this operation", after signing in with Google, please try the following: Go to live server extension -> Settings:Host -> Change the default 127.0.0.1 to localhost

**Comment by marcinwmichna** [comment]
@buigabor seems to work! Thanks 👍

**Comment by jrobind** [comment]
Thanks for raising this @marcinwmichna. I've actually made some changes to the project.

By default it will now work without Firebase. Lesson notes are now persisted to the browser's local storage. I've done this so it will be easier for contributors (especially first timers) to get the project running locally. Currently, the project will still work with Firebase but you will need to create your own Firebase project and add your configuration.

I've updated the docs to reflect these changes too.

When you fetch the latest changes the project should run without having to authenticate using Google. I'll leave this issue open for the time being!
